src/herder/Upgrades.cpp

```diff
@@ -163,7 +227,8 @@ Upgrades::getParameters() const
 }
 
 std::vector<LedgerUpgrade>
-Upgrades::createUpgradesFor(LedgerHeader const& header) const
+Upgrades::createUpgradesFor(LedgerHeader const& header,
```

Comment:
That's true, but `HerderImpl` passes in `lcl.header` instead of using ltx, which is why I kept it like this.

Reply:
Interesting, this actually might make subtle difference: in case of protocol version upgrade `lcl.header` is always `vCurr`, while IIUC `ltx.header` can become `vNext`. If that's intended, probably we should rename this to `lclHeader` for clarity and add a comment, because someone might 'optimize' this in the future.

Reply:
Updated to lclHeader.